test/Broadway/EventSourcing/EventSourcingRepositoryTest.php

```diff
@@ -78,6 +81,34 @@ public function it_throws_an_exception_if_the_static_method_does_not_exist()
         $repository->load('y0l0');
     }
 
+    /**
+     * @test
+     */
+    public function it_publishes_decorated_events()
```

Comment:
Can you move this test to the base class? I think the base class exists for historical reasons, but I'd rather have the test there for now.